packages/router/src/models.ts

```diff
@@ -139,7 +138,7 @@ export interface DefaultExport<T> {
  * @see [Route.loadChildren](api/router/Route#loadChildren)
  * @publicApi
  */
-export type LoadChildrenCallback = () => Type<any>|NgModuleFactory<any>|Routes|
+export type LoadChildrenCallback = () => |Type<any>|NgModuleFactory<any>|Routes|
```

Comment:
```suggestion
export type LoadChildrenCallback = () => Type<any>|NgModuleFactory<any>|Routes|
```
(there are also similar changes in other files below)